src/stores/breadcrumb.ts

```diff
@@ -37,7 +37,7 @@ interface BreadcrumbActions {
 
 export type BreadcrumbStore = Store<string, BreadcrumbState, BreadcrumbGetters, BreadcrumbActions>;
```

Comment:
todo: instead of this type it should be possible to use the following inside of `c-breadcrumb.vue`:

```
interface Setup {
    breadcrumbStore: ReturnType<typeof breadcrumbStore>;
    addContextPathToUrl: typeof addContextPathToUrl;
  }
```